api.ts

```diff
@@ -1583,6 +1610,18 @@ export function makeSyncTable<
       continuation,
     } as SyncFormulaResult<K, L, SchemaT>;
   };
+  const executeUpdate = wrappedExecuteUpdate ? async function execUpdate(
+    params: ParamValues<ParamDefsT>,
+    updates: Array<SyncUpdate<K, L, SchemaDefT>>,
+    context: SyncExecutionContext,
+  ) {
```

Comment:
If you add an explicit return type here, does that help with the `any`?

Reply:
Unfortunately, no:
```
Type 'ObjectSchemaDefinitionType<K, L, SchemaT>' is not assignable to type 'Partial<{ [K in keyof SchemaWithNoFromKey<SchemaDefT>]: SchemaType<SchemaWithNoFromKey<SchemaDefT>[K]>; }>'
```